<launch>
  <!-- Variable names -->
  <arg name="camera_node" default="mv_bluefox_camera_node" />
  <arg name="calibration_file" default="" />
	
  <node pkg="mv_bluefox_driver" type="camera_node" name="$(arg camera_node)" output="screen" >
    <param name="width" type="int" value="752" />
    <param name="height" type="int" value="480" />
    <param name="auto_gain" type="bool" value="true" />
    <param name="use_color" type="bool" value="true" /> 
    <param name="frame_id" type="string" value="bluefox" />
    <param name="calibration_file" type="string" value="$(arg calibration_file)" />
    
  </node>
</launch>
